overrides/windows-override.json

```diff
@@ -1516,6 +1516,31 @@
                             }
                         ]
                     },
+                    {
+                        "condition": {
+                            "preview": true
+                        },
+                        "patchSettings": [
+                            {
+                                "op": "replace",
+                                "path": "/additionalCheck",
+                                "value": "enabled"
+                            }
+                        ]
+                    },
+                    {
+                        "condition": {
+                            "minSupportedVersion": "0.134.4",
+                            "maxSupportedVersion": "0.134.4"
```

Comment:
Need to confirm what the exact version number for the upcoming Windows preview release is.